src/OrbitGl/CaptureWindow.cpp

```diff
@@ -120,8 +120,9 @@ void CaptureWindow::MouseMoved(int x, int y, bool left, bool right, bool middle)
   if (left && !picking_manager_.IsDragging() && !app_->IsCapturing()) {
     Vec2i mouse_click_screen = viewport_.WorldToScreen(mouse_click_pos_world_);
     Vec2 mouse_pos_world = viewport_.ScreenToWorld({x, y});
-    time_graph_->SetVerticalScrollingOffset(timegraph_click_scrolling_offset_ +
-                                            mouse_click_pos_world_[1] - mouse_pos_world[1]);
+    orbit_gl::TrackContainer* track_container = time_graph_->GetTrackContainer();
```

Comment:
This change is adding a simple refactor as timegraph_click_scrolling_offset is the previous VerticalScrollingOffset. As it was a few lines, I allow myself to include it here.